package.json

```diff
@@ -2,19 +2,16 @@
   "name": "use-axios-client",
   "version": "2.0.0",
   "description": "Make axios requests in React using hooks.",
-  "sideEffects": false,
-  "main": "bin/index.js",
-  "types": "bin/index.d.ts",
   "scripts": {
-    "build": "webpack --config webpack.config.js",
+    "build": "pika build",
     "coverage": "yarn test --coverage --watchAll=false && yarn report",
-    "lint": "eslint src webpack.config.js babel.config.js --ext js,jsx,ts,tsx",
+    "lint": "eslint src babel.config.js --ext js,jsx,ts,tsx",
     "prettier": "prettier --write **/*.{js,jsx,ts,tsx}",
     "report": "cd coverage/lcov-report && open index.html",
     "test": "jest --watch",
     "test:once": "jest",
-    "tsc": "tsc --emitDeclarationOnly",
-    "prepare": "yarn lint && yarn test:once && yarn tsc && yarn build"
+    "tsc": "tsc --noEmit",
+    "prepare": "yarn lint && yarn test:once && yarn tsc && yarn build && cd pkg"
```

Comment:
What's the need for this `cd pkg`?

Reply:
Is this the script that is run right before publishing? If so, you'll need to `cd pkg` before the publish.

Reply:
Ahhh I see, I just realized we're getting rid of the `main` entry so that makes sense. If this makes linking easier, I'm all for it.